django_readers/projectors.py

```diff
@@ -26,7 +27,10 @@ def relationship(name, related_projector):
     """
 
     def value_getter(instance):
-        related = attrgetter(name)(instance)
+        try:
+            related = attrgetter(name)(instance)
+        except ObjectDoesNotExist:
+            return None
         return map_or_apply(related, related_projector)
```

Comment:
This divided opinion I know but I'd put the return map or apply line inside the try block to show the happy path tidily then handle the exception

Reply:
But if `map_or_apply` raises an `ObjectDoesNotExist` that probably means something different and so I don't want that to be caught..